.github/workflows/main_win.yaml

```diff
@@ -154,6 +160,43 @@ jobs:
         uses: ./.github/actions/docker_win
         with:
           service: ${{ needs.initialize.outputs.docker_runner_service }}
+
+  # Builds, tags, and pushes Cobalt unit test image to ghr.
+  docker-unittest-image:
+    if: always()
```

Comment:
Why does it need to always run, I think this is needed only if on-device tests are being run?

Reply:
Actually, since we will be running ODT job directly on the runner we can remove docker-unittest-image job altogether.